packages/rpc-graphql/src/__tests__/transaction-tests.ts

```diff
@@ -895,6 +895,108 @@ describe('transaction', () => {
                     },
                 });
             });
+
+            it('withdraw-withheld-tokens-from-mint', async () => {
+                expect.assertions(1);
+                const source = /* GraphQL */ `
+                    query testQuery($signature: Signature!) {
+                        transaction(signature: $signature) {
+                            message {
+                                instructions {
+                                    programId
+                                    ... on SplTokenWithdrawWithheldTokensFromMint {
+                                        mint {
+                                            address
+                                        }
+                                        feeRecipient {
+                                            address
+                                        }
+                                        withdrawWithheldAuthority {
+                                            address
+                                        }
+                                    }
+                                }
+                            }
+                        }
+                    }
+                `;
+
+                const result = await rpcGraphQL.query(source, { signature });
+                expect(result).toMatchObject({
+                    data: {
+                        transaction: {
+                            message: {
+                                instructions: expect.arrayContaining([
+                                    {
+                                        feeRecipient: {
+                                            address: expect.any(String),
+                                        },
+                                        mint: {
+                                            address: expect.any(String),
+                                        },
+                                        programId: 'TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb',
+                                        withdrawWithheldAuthority: {
+                                            address: expect.any(String),
+                                        },
+                                    },
+                                ]),
```

Comment:
If you include the `signers` field in this test, you can just include both payloads in the check!

```suggestion
                                instructions: expect.arrayContaining([
                                    {
                                        feeRecipient: {
                                            address: expect.any(String),
                                        },
                                        mint: {
                                            address: expect.any(String),
                                        },
                                        programId: 'TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb',
                                        withdrawWithheldAuthority: {
                                            address: expect.any(String),
                                        },
                                        signers: null,
                                    },
                                    {
                                        feeRecipient: {
                                            address: expect.any(String),
                                        },
                                        mint: {
                                            address: expect.any(String),
                                        },
                                        programId: 'TokenzQdBNbLqP5VEhdkAS6EPFLC1PHnBqCXEpPxuEb',
                                        withdrawWithheldAuthority: {
                                            address: expect.any(String),
                                        },
                                        signers: expect.arrayContaining(expect.any(String)),
                                    },
                                ]),
```